packages/theme-chalk/src/menu.scss

```
@@ -58,7 +58,10 @@
   padding-left: 0;
   background-color: getCssVar('menu-bg-color');
   box-sizing: border-box;
-
+  & .#{$namespace}-menu-item:not(.is-disabled):hover {
+    outline: none;
+    background-color: getCssVar('menu-hover-bg-color');
+  }
```

Styles will be broken on keyboard access.

```suggestion
  & .#{$namespace}-menu-item:not(.is-disabled):hover
  & .#{$namespace}-menu-item:not(.is-disabled):focus-visible {
    outline: none;
    color: getCssVar('menu-hover-text-color');
    background-color: getCssVar('menu-hover-bg-color');
  }
```

---

Has it been fixed?, now I have this problem too